### DISCUSSION TITLE
230710 FE 번개 스터디 / module system (12:00 ~ 13:00)

### DISCUSSION THREAD

**Discussion by ukkodeveloper** [woowacourse-teams/2023-shook#23] [스터디]
## 회의 안건
모듈 시스템을 공부한 내용에 대해  이야기를 나눈다. 목적은 모듈 시스템을 이해하여 보다 수월하게 프론트엔드 환경을 설정하는 것이다.

**Comment by jariita** [comment]
https://cruelladevil.notion.site/6c4e886ef5654211a9823a1e17fd9e12?pvs=4
